src/super_gradients/common/plugins/deci_client.py

```diff
@@ -207,12 +218,18 @@ def add_model(
         self.lab_client.add_model_v2(model_metadata=model_metadata, hardware_types=hardware_types, model_path=model_path, model=model, **kwargs)
 
 
-def _load_cfg(config_path: str) -> DictConfig:
-    """Load a hydra config file.
-    :param config_path: Full path of the hydra config file.
-    :return:            Hydra config instance"""
-    GlobalHydra.instance().clear()
+def _move_file_to_folder(src_file_path: str, dest_dir_name: str) -> str:
+    """Move a file to a newly created folder in the same directory.
+
+    :param src_file_path:   Path of the file to be moved.
+    :param dest_dir_name:   Name of the destination folder.
+    :return:                The path of the moved file.
+    """
+    src_dir_path = os.path.dirname(src_file_path)
+
+    dest_dir_path = os.path.join(src_dir_path, dest_dir_name)
+    dest_file_path = os.path.join(dest_dir_path, os.path.basename(src_file_path))
 
-    arch_params_file = Path(config_path)
-    with hydra.initialize_config_dir(config_dir=str(arch_params_file.parent), version_base=None):
-        return hydra.compose(config_name=arch_params_file.name)
+    os.makedirs(dest_dir_path, exist_ok=True)
```

Comment:
Same, maybe `wait_for_master` here?

Reply:
Because evert process has its own file (see comment above) there is no point to `wait_for_master`, right?